src/components/Editor/Editor.js

```diff
@@ -30,21 +29,21 @@ export default function Editor() {
         <label>Alignment</label>
         <div className="radio-group">
           <label>
-            <input name="align" type="radio" value="left" />
+            <input name="align" type="radio" value="left" onClick={() => setTextFormat('left')} />
```

Comment:
With the way you implemented the onClick event handler it makes having `value="left"` on this input unnecessary since it isn't being used for anything. 

The way it is now is fine, but alternatively you could use `setTextFormat(e.target.value)` in the onClick.